scripts/blender/nerfstudio_addon_1_0.py

```diff
@@ -0,0 +1,404 @@
+bl_info = {
+    "name": "Nerfstudio Add-On",
```

Comment:
Add an empty file `scripts/blender/__init__.py`
This will cause the linter to parse it. Then run `ns-dev-test`, you will likely see many lint issues. These should be resolved, sometime they can be a bit tricky, feel free to let me know if you have questions.